RevenueCat/Plugins/iOS/PurchasesUnityHelper.m

```diff
@@ -557,3 +567,21 @@ void _RCSetCreative(const char *creative) {
 void _RCCollectDeviceIdentifiers() {
     [_RCUnityHelperShared() collectDeviceIdentifiers];
 }
+
+void _RCCanMakePayments(const char *featuresJSON) {
+    NSError *error = nil;
+    
+    NSDictionary *canMakePaymentsRequest = [NSJSONSerialization 
+        JSONObjectWithData:[convertCString(featuresJSON)        
+            dataUsingEncoding:NSUTF8StringEncoding]
+        options:0
+        error:&error];
```

Comment:
The formatting of these lines is a little bit off

Reply:
there seemed to be a really wide array of formattings for this kinda call... this felt the most readable to me, but open to suggestions.

Reply:
In objc, `:` are aligned if the parameters are in different lines. Apart from that, maybe extracting the data into a variable also helps. Something like this :

```
NSData *data = [convertCString(featuresJSON) dataUsingEncoding:NSUTF8StringEncoding];
NSDictionary *canMakePaymentsRequest = [NSJSONSerialization JSONObjectWithData:data
                                                                       options:0
                                                                         error:&error];
```